packages/mui-joy/src/Alert/Alert.tsx

```diff
@@ -124,6 +124,9 @@ const Alert = React.forwardRef(function Alert(inProps, ref) {
     size = 'md',
     startDecorator,
     endDecorator,
+    component,
+    slots = {},
+    slotProps = {},
```

Comment:
@hbjORbj What's the reason to explicitly declare these props and compose them again in the `externalForwardedProps`?

My intention is that the `useSlot` will take care of them internally so we don't have to do this in all components.

Reply:
If we don't do this, then they don't show up in Proptypes (and hence not in JSON files), and hence they don't appear in Prop table in API docs